VideoLocker/src/main/java/org/edx/mobile/social/google/GoogleOauth2.java

```diff
@@ -231,10 +244,17 @@ public String getEmail(){
     }
 
     private String getScopes() {
+
+
         String login = "https://www.googleapis.com/auth/plus.login";
 //      String email = "https://www.googleapis.com/auth/plus.profile.emails.read";
+
+        //TODO-  this scope is deprecated
+        //see document - https://developers.google.com/+/api/oauth
```

Comment:
@aleffert as it is an existing code,  i dont think we have enough time for the testing.  sure we will update it later.